Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Button.kt

```diff
@@ -123,7 +123,7 @@ private fun ButtonPreview() {
 private fun RectangleButtonPreview() {
     JetsnackTheme {
         JetsnackButton(onClick = {}, shape = RectangleShape) {
-            Text(text = "Demo")
-        }
+        Text(text = "Demo")
+    }
```

Comment:
Revert back?

Reply:
This is actually done by SpotlessApply. Without it the build fails. I'll find or raise a bug for ktlint or spotless.

Reply:
Found it! https://github.com/pinterest/ktlint/issues/1221

Reply:
Cool! Should we revert the version update back until it's fixed?

Reply:
Done